src/datasources/redis/pmseries_srv.ts

```diff
@@ -89,6 +93,31 @@ class PmSeriesApi {
         return response.data;
     }
 
+    async labelsNames(): Promise<string[]> {
```

Comment:
Two plurals seem unusual here, could you name it `labelNames` and `getLabelNames`?
I guess `labelsValues` is fine, as we want the values of multiple labels.